src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -41,6 +41,7 @@ public sealed class AssemblyModel
 
         internal DataTable GetStatistics()
         {
+            //fixme. Column names should be defined as constant methods
```

Comment:
Before merging, this should either be fixed, or an issue filed and the comment removed.